Fix modal verb "must" (#4222)
cplusplus/draft: source/expressions.tex

no new exception object is created.
The exception is no longer considered to be caught.
\begin{example}

> Code that is executed because of an exception, but cannot
> completely handle the exception itself, can be written like this:

Reviewer: I don't like this phrasing (before or after the "must" fix). I think this would be good enough for now:
```suggestion
An exception handler that cannot completely handle the exception itself
can be written like this:
```

Author: done
